Tutorials/01_Psi4NumPy-Basics/1b_psi4_numpy_datasharing.ipynb

```diff
@@ -4,9 +4,9 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Psi4-NumPy Datasharing\n",
+    "# Psi4 $\\leftrightarrow$ NumPy Data Sharing\n",
     "\n",
-    "The heart of the Psi4NumPy project its the ability to easily share and manipulate quantities in Python. While Psi4 offers the ability to manipulate most objects and perform tensor operations at the Python layer, it is often much easier to use the NumPy project as its focus is on ease of use rather than optimal performance. Fortunately, Psi4 offers seemless integration with the NumPy framework. More details on the underlying functions can be found in the Psi4 [documentation](http://psicode.org/psi4manual/master/numpy.html).\n",
+    "The heart of the Psi4NumPy project its the ability to easily share and manipulate quantities in Python. While Psi4 offers the ability to manipulate most objects and perform tensor operations at the Python layer, it is often much easier to use the NumPy project, as its focus is on ease of use rather than optimal performance. Fortunately, Psi4 offers seemless integration with the NumPy framework. More details on the underlying functions can be found in the Psi4 [documentation](http://psicode.org/psi4manual/master/numpy.html).\n",
```

Comment:
Hmm, caught the "seemless" in the paper. Guess I missed this one. You can add it to your PR through the edit button on the website if you like, Dom. Otherwise, as you say, will catch later.